src/components/Admin/resources/competition/series/SeriesCreate.tsx

```diff
@@ -1,19 +1,25 @@
 import {FC} from 'react'
-import {BooleanInput, FormTab, ReferenceInput, required, SelectInput, TabbedForm, TextInput} from 'react-admin'
+import {AutocompleteInput, FormTab, NumberInput, ReferenceInput, required, TabbedForm} from 'react-admin'
 
 import {MyCreate} from '@/components/Admin/custom/MyCreate'
 import {MyDateTimeInput} from '@/components/Admin/custom/MyDateTimeInput'
 
 export const SeriesCreate: FC = () => (
-  <MyCreate>
+  <MyCreate
+    transform={(record) => {
+      // automaticky sa na BE posiela cely record, ale BE read_only (aj neexistujuce) fieldy ignoruje
+      // radsej z payloadu odstranime aspon sety
+      delete record.problems
```

Comment:
je to treba pri Create? mozno je to len pri Edite, kedze vychadzas z existujuceho objektu.. pri Create asi nemame tento `problems` field, ak nanho nemame input

Reply:
je to debug step ktorý som zabudol odstrániť